parsers/AW.py

```diff
@@ -55,17 +55,17 @@ def fetch_production(
     missing_from_total = reported_total - sources_total
     unknown = missing_from_total if missing_from_total > 0 else 0
     # We're using Fossil data to get timestamp in correct time zone
-    local_date_time = datetime.strptime(fossil["timestamp"], "%Y-%m-%d %H:%M:%S.%f")
+    local_date_time = datetime.strptime(biogas["timestamp"], "%Y-%m-%d %H:%M:%S.%f")
     zone_date_time = arrow.Arrow.fromdatetime(local_date_time, "America/Aruba")
 
     data = {
         "zoneKey": zone_key,
         "datetime": zone_date_time.datetime,
         "production": {
-            "oil": float(fossil["value"]),
-            "wind": float(wind["value"]),
-            "solar": float(solar["value"]),
-            "biomass": float(biogas["value"]),
+            "oil": round(float(fossil["value"]), 3),
```

Comment:
Never really been fond of discarding data when it's not necessary so I'm biased, but is there a point in doing this? If the backend still stories it as floats, it won't save anyone storage space (assuming no compression), and if it doesn't, it'll do away with the extras precision by itself when converting it to whatever data type the backend uses. 

By the way, if the point is to get rid of floating point errors, rounding to 13 decimal points will suffice.

Reply:
Are you using `poetry run test_parser AW` for example? That should give you a pretty printed output.